server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/PreCheckRuntimeFlowableTask.java

```diff
@@ -299,7 +299,8 @@ private void doSqlCheckAndDatabasePermissionCheck(TaskEntity preCheckTaskEntity,
                 }
             }
             unauthorizedDatabases =
-                    databaseService.filterUnauthorizedDatabases(schemaName2PermissionTypes, connectionConfig.getId());
+                    databaseService.filterUnauthorizedDatabases(schemaName2PermissionTypes, connectionConfig.getId(),
```

Comment:
There is also a class named PreCheckRuntimeFlowableTaskCopied. Does it also need to be modified?

Reply:
Confirmed with @smallsheeeep , `PreCheckRuntimeFlowableTaskCopied` is not in use currently.